app/src/pages/playground/playgroundUtils.ts

```diff
@@ -342,6 +345,21 @@ export function getModelInvocationParametersFromAttributes(
   };
 }
 
+export function getResponseFormatFromAttributes(parsedAttributes: unknown) {
+  const { success, data } =
+    modelConfigWithResponseFormatSchema.safeParse(parsedAttributes);
+  if (!success) {
+    return {
+      responseFormat: undefined,
+      parsingErrors: [MODEL_CONFIG_WITH_RESPONSE_FORMAT_PARSING_ERROR],
+    };
+  }
+  return {
+    responseFormat: data.llm.invocation_parameters.response_format,
+    parsingErrors: [],
+  };
+}
+
```

Comment:
haven't gotten tot he schema yet, will this fail if there is no response format at all? or just if it's not the right shape? We shouldn't show parsing errors if it's not present

Reply:
Only if it is present and not the right shape. Added a test to confirm.